devTools/core/src/Command/Util/TranslationTestTool.php

```diff
@@ -56,19 +54,19 @@ private function addTranslations(string $language, string $groupName): void
         $yml2 = Yaml::parseFile($filepath2);
         $langStrings = array_shift($yml2);
         foreach ($langStrings as $langString) {
-            $sourceObj = new TransUnit($langString['value'], 'tr_' . $langString['value']);
+            $sourceObj = new TranslationUnit('tr_' . $langString['value'], null, $langString['value'], );
             $this->saveTranslationRecord($groupName, $sourceObj, $language);
         }
     }
 
     /**
      * @param string $groupName
-     * @param TransUnit $source
+     * @param TranslationUnit $source
      * @param string $language
      * @return void
      * @throws Exception
      */
-    private function saveTranslationRecord(string $groupName, TransUnit $source, string $language): void
+    private function saveTranslationRecord(string $groupName, TranslationUnit $source, string $language): void
```

Comment:
using `OrangeHRM\Installer\Util\V1\Dto\TransUnit` here is fine, no need to create another class `TranslationUnit`

Reply:
The classes are a bit different, In TranslationUnit I need the to have the source as one of its attributes as the target is generated using the source.